src/forms/button.test.jsx

```diff
@@ -18,83 +12,62 @@ describe('Button', () => {
 		});
 	});
 
-	it('applies variant classes for each variant prop', () => {
-		const variants = ['primary', 'fullWidth', 'small', 'bordered'];
+	it('has a default state', () => {
+		const button = mount(<Button>Click Me</Button>);
```

Comment:
Slightly more DRY approach would be to define a `renderComponent` function:

```
const renderComponent = (props = {}) => mount(<Button {...props}>Click Me</Button>);

describe('Button', () => {
...

   const button = renderComponent({primary:true});
})
```

Reply:
I prefer a little repetition here to keep the react api in the test definition. This prevents looking around for the definition of the `renderComponent` when you want to change.